web/Makefile

```diff
@@ -91,6 +91,11 @@ devel-install: $(WEBPACK_TEST)
 	rm -f ~/.local/share/cockpit/$(PACKAGE_NAME)
 	ln -s `pwd`/dist ~/.local/share/cockpit/$(PACKAGE_NAME)
 
+custom-install: $(WEBPACK_TEST)
+	mkdir -p ~/.local/share/cockpit
+	rm -f ~/.local/share/cockpit/$(PACKAGE_NAME)
+	ln -s `pwd`/dist ~/.local/share/cockpit/$(PACKAGE_NAME)
```

Comment:
what is purpose of this? It looks same as devel-install

Reply:
Yes, it is just another name for `devel-install`. Why?

Reply:
sorry ... some experiment which i forgot to remove